jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -46,10 +51,9 @@ def json_changebase(obj, changer):
 # Transaction serialization and deserialization
 
 def deserialize(tx):
```

Comment:
This function is turning out to be especially annoying to deal with and could use review.